components/application-mgt/org.wso2.carbon.identity.application.mgt/src/test/resources/dbscripts/identity.sql

```diff
@@ -796,10 +796,20 @@ CREATE TABLE IF NOT EXISTS IDN_RECOVERY_DATA (
   STEP VARCHAR(127) NOT NULL,
   TIME_CREATED TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
   REMAINING_SETS VARCHAR(2500) DEFAULT NULL,
+  RECOVERY_FLOW_ID VARCHAR(255),
   PRIMARY KEY(USER_NAME, USER_DOMAIN, TENANT_ID, SCENARIO,STEP),
   UNIQUE(CODE)
 );
 
+CREATE TABLE IF NOT EXISTS IDN_RECOVERY_FLOW_DATA (
+    RECOVERY_FLOW_ID VARCHAR(255) NOT NULL,
+    CODE VARCHAR(255),
+    ATTEMPT INTEGER DEFAULT 0 NOT NULL,
```

Comment:
shall we have this as failed_attempts?

Reply:
Updated with 4d7ce5da680764fe2b585a605ba39a42358a0f92